src/sagemaker/image_uri_config/huggingface-neuron.json

```diff
@@ -0,0 +1,45 @@
+{
+    "inference": {
+        "processors": ["inf"],
+        "version_aliases": {"4.12": "4.12.3"},
+        "versions": {
+            "4.12.3": {
+                "version_aliases": {"pytorch1.9": "pytorch1.9.1"},
+                "pytorch1.9.1": {
+                    "py_versions": ["py37"],
+                    "repository": "huggingface-pytorch-inference-neuron",
+                    "registries": {
+                        "af-south-1": "626614931356",
+                        "ap-east-1": "871362719292",
+                        "ap-northeast-1": "763104351884",
+                        "ap-northeast-2": "763104351884",
+                        "ap-northeast-3": "364406365360",
+                        "ap-south-1": "763104351884",
+                        "ap-southeast-1": "763104351884",
+                        "ap-southeast-2": "763104351884",
+                        "ca-central-1": "763104351884",
+                        "cn-north-1": "727897471807",
+                        "cn-northwest-1": "727897471807",
+                        "eu-central-1": "763104351884",
+                        "eu-north-1": "763104351884",
+                        "eu-west-1": "763104351884",
+                        "eu-west-2": "763104351884",
+                        "eu-west-3": "763104351884",
+                        "eu-south-1": "692866216735",
+                        "me-south-1": "217643126080",
+                        "sa-east-1": "763104351884",
+                        "us-east-1": "763104351884",
+                        "us-east-2": "763104351884",
+                        "us-gov-west-1": "442386744353",
+                        "us-iso-east-1": "886529160074",
+                        "us-west-1": "763104351884",
+                        "us-west-2": "763104351884"
+                    },
+                    "container_version": {"inf": "ubuntu18.04"},
+                    "image_versions": ["v1.0"],
```

Comment:
Does this refer to -v1.0 in 763104351884.dkr.ecr.us-west-2.amazonaws.com/huggingface-pytorch-inference-neuron:1.9.1-transformers4.12.3-neuron-py37-sdk1.17.1-ubuntu18.04-v1.0? if so we should remove this. The Image has multiple available tags see: https://github.com/aws/deep-learning-containers/releases/tag/v1.0-hf-4.12.3-pt-1.9.1-inf-neuron-sdk1.17.1-py37 one of them is 1.9.1-transformers4.12.3-neuron-py37-sdk1.17.1-ubuntu18.04 (just without the v1.0).
The v1.0 indicates the version of the container and the assimov team might update the container to apply security patches then we would need update this as well its a better practice to always use the latest version through 1.9.1-transformers4.12.3-neuron-py37-sdk1.17.1-ubuntu18.04 tag and remove repo_version

Reply:
removed.